doc(cache): Documentation for cd caching agents config

german-muzquiz opened the pull request:
Resolves issue: [ENG-5775]



[ENG-5775]: https://armory.atlassian.net/browse/ENG-5775

Commit:
doc(cache): Documentation for cd caching agents config

german-muzquiz committed:
Merge branch 'master' into doc/caching-agents

ncknt commented on content/en/docs/armory-admin/caching-agents.md:
```diff
@@ -0,0 +1,43 @@
+---
+title: Configuring clouddriver caching agents
```
Capitalize title

ncknt commented on content/en/docs/armory-admin/caching-agents.md:
```diff
@@ -0,0 +1,43 @@
+
+|Key|Description|Default Value|
+|----|---|---|
+|sql.scheduler.enabled|Enable or disable the sql scheduler|false
```
Wrap options in backticks? That's the style I used for agent options so it'd be nice to be consistent.

ncknt commented:
I'd add a note that the `sql.connectionPools` needs to be configured with a link to https://docs.armory.io/docs/armory-admin/clouddriver-sql-configure/

Ideally the page would also include what metrics to check to know when to scale horizontally (or a link to a page that mentions it). It can obviously be a later improvements but if you already have such a page, link it!

You could also link to this blog post that goes over caching agents: https://www.armory.io/blog/deep-dive-into-clouddriver/ (the author is amazing by the way!)

Commit:
doc(cache): Reference to other docs

Commit:
Merge branch 'doc/caching-agents' of github.com:armory/docs into doc/caching-agents

german-muzquiz committed:
Merge branch 'master' into doc/caching-agents

aimeeu committed:
Add page for Clouddriver caching agent concept

New page with content from Nico's 2019 blog post.
Renamed caching-agents to caching-agents-config.

aimeeu commented:
Talked to Nico. Created a new page for caching agents concepts based on Nico's 2019 blog post.  Renamed files. Linked concepts and config.
Will do another pass on Monday, March 1.

aimeeu committed:
change initial heading wording

aimeeu committed:
fixed linting issues

317brian committed:
copy edits and remove netflix example

317brian committed:
copyedits

317brian committed:
add tm

aimeeu committed:
Merge branch 'master' into doc/caching-agents

aimeeu committed:
fix more linting issues

aimeeu committed:
still more linting
